doc/endpoints.md

```diff
@@ -14,7 +14,8 @@ The authentication proxy behaves in the same way as in case of [WMS](#web-map-se
 
 The WFS proxy parses request and adapts it in few ways in case of WFS-T 1.0, 1.1 and 2.0:
 - If incoming WFS-T request Insert, Replace, or Update refers to an attribute, that does not exist yet in DB, the attribute is automatically created in DB table before redirecting WFS-T request to GeoServer. Data type of the attributes is `VARCHAR(1024)`. Also if QML style is used, attribute is automatically added to QGS project file. If attribute creation fails for any reason, warning is logged and request is forwarded to GeoServer nevertheless.
-- Bounding box of each layer referenced in incoming Insert, Replace, Update, or Delete WFS-T request is updated in asynchronous task after the request is finished. 
+- Bounding box and thumbnail of each layer referenced in incoming Insert, Replace, Update, or Delete WFS-T request is updated in [asynchronous chain](async-tasks.md) after the request is finished.
+- Calling WFS-T on a layer when previous WFS-T asynchronous chain is still running causes re-run of WFS-T asynchronous chain after the current one is finished.
```

Comment:
Mention also concurency behavior of WFS-T asynchronous chain when PATCH/POST is already running